pkg/utils/host/host.go

```diff
@@ -175,3 +182,17 @@ func GetProcCmdline(pid int) []string {
 	cmdlineBytes, _ := os.ReadFile(filepath.Join(HostProcFs, pidStr, "cmdline"))
 	return strings.Split(string(cmdlineBytes), "\x00")
 }
+
+func GetProcStat(pid int) (ProcStat, error) {
+	pidStr := fmt.Sprint(pid)
+	var stat syscall.Stat_t
+	if err := syscall.Stat(filepath.Join(HostProcFs, pidStr), &stat); err != nil {
+		return ProcStat{}, fmt.Errorf("reading /proc/%d/stat: %w", pid, err)
```

Comment:
The path in the error message does not match the path used in the `Stat()` call (`/host` prefix and `stat` suffix)
```suggestion
	path := filepath.Join(HostProcFs, fmt.Sprint(pid))
	if err := syscall.Stat(path, &stat); err != nil {
		return ProcStat{}, fmt.Errorf("reading %s: %w", path, err)
```